src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -389,6 +389,12 @@ public Element createElement(org.w3c.dom.Element element, Element parent) throws
         }
 
       case Div.MESSAGEML_TAG:
+        String clazz = element.getAttribute(CLASS_ATTR);
+        if(SplittableElement.SPLITTABLE_GROUPS.contains(clazz)){
+          // Special div created by a splittable element, it is not converted in MessageML
+          // as it exist only in PresentationML
+          return null;
```

Comment:
When converting from PresentationML to MessageML, in some cases there is a div wrapper to remove:

```xml
<div class="element-implementing-splittable-group">
...
   <presentationMLelement>value</presentationMLelement>
 </div>
```

```xml
<messageMLelement attribute=...>value</messageMLelement >
```

The divs to be dropped are identified by reading the class attribute ("textfield-group", "textarea-group", "dropdown-group", "person-selector-group")
The problem is that this list must be maintained and it is not so reliable. A suggest a specification modification for
the div wrapper in presentationML, by adding a very specific attribute